bsp/boards/iot-lab_M3/leds.c

```diff
@@ -136,7 +136,9 @@ uint8_t leds_radio_isOn()
 void leds_debug_on() {}
 void leds_debug_off() {}
 void leds_debug_toggle() {}
-uint8_t leds_debug_isOn() {}
+uint8_t leds_debug_isOn() {
+  return 0;

```

Comment:
Done. https://github.com/openwsn-berkeley/openwsn-fw/pull/313/commits/7642763f90f7cee279e19ad15420a352fe8c2c3d
and 8d174e6